dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/relationship/RequestParams.java

```diff
@@ -57,6 +60,18 @@ class RelationshipCriteria extends PagingAndSortingCriteriaAdapter
 
     private Class<?> identifierClass;
 
+    private List<FieldPath> fields = RelationshipsExportController.DEFAULT_FIELDS_PARAMS;
```

Comment:
To be consistent with `enrollment` and `event` above I would use lombok `@Getter` and `@Setter`